### ISSUE TITLE
Reformatting breaks null coalescing

### ISSUE THREAD

**Issue by Holzhaus** [qarmin/qml_formatter#4]
When reformatting code that uses JavaScripts [null coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator), this leads to a syntax error.

```udiff
-            property string label: shortName ?? name
+            property string label: shortName ? ? name
```

**Comment by qarmin** [comment]
Fixed by 80109bfc7a697e5fe9c5e5e4c6c9388b464eef4b

**Comment by Holzhaus** [comment]
Wow, that was quick. Thanks.

Unfortunately the fix does not work in all cases:

1. This works fine:
   ```udiff
   -            property string label: shortName   ?? name
   +            property string label: shortName ?? name
   ```

2. This results in a syntax error:
   ```udiff
   -            property string label: shortName?? name
   +            property string label: shortName ? ? name
   ```
3. And this  is not reformatted at all:
   ```udiff
                property string label: shortName ??name
   ```

**Comment by qarmin** [comment]
Fixed again by 64745f0d83f2c99aeae1eb56c50c9a244db82081
